app/views/layouts/_navbar.html.erb

```diff
@@ -59,6 +59,11 @@
 						<%= link_to "Account", edit_user_path(current_user), :title => "View your account info" %>
 					</li>
 					<% end %>
+					<% if @course and @cud %>
+					<li role="presentation">
+						<%= link_to "Course Profile", course_course_user_datum_path(@course, @cud), :title => "View your user profile for the current course" %>
```

Comment:
Run 'rake routes' to get a list of routes in Autolab, as well as helper functions to create those routes, when they are available.  course_course_user_datum_path is indeed one of the paths defined by our route schema.  For what it's worth, you could have also done [@course, @cud] in place of the function call and got the same result.